src/main/java/com/team4/testingsystem/dto/AnswerDTO.java

```diff
@@ -0,0 +1,53 @@
+package com.team4.testingsystem.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.team4.testingsystem.entities.Answer;
+
+import java.util.Objects;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class AnswerDTO {
+    private String answer;
+    private Boolean correct;
```

Comment:
Are we gonna use this DTO inside TestDTO and send correct answers it to the user?)

Reply:
the testDTO uses a questionDTO in which the value of the correct field is not passed on creation
it will looks like 
![image](https://user-images.githubusercontent.com/75488040/128219105-3fd65ada-12ed-4151-9512-7c1f04017bdd.png)

Reply:
We can get the info on creating the answers. And sometimes (but not always), we will need to send the correct flag as well

Reply:
Why do we need available flag in dto?

Reply:
maybe not needed)